documentation/src/main/asciidoc/ch02.asciidoc

```diff
@@ -785,6 +785,12 @@ Hibernate Validator!
                 Pessoa Fídsica)|Brazil|None
 |`@TituloEleitoral`|`CharSequence`|Checks that the annotated character sequence represents
                 a Brazilian voter ID card number (link:$$http://ghiorzi.org/cgcancpf.htm$$[Título Eleitoral])|Brazil|None
+|`@NIP`|`CharSequence`|Checks that the annotated character sequence represents a Polish VAT identification
+ 				number ((link:$$https://pl.wikipedia.org/wiki/NIP$$[NIP]))|Poland|None
```

Comment:
The best (and only) interesting piece of information I found is this one: http://www.exceldoseujeito.com.br/2008/12/19/validar-cpf-cnpj-e-titulo-de-eleitor-parte-ii/ . Btw, I only see one commit for the typo, did you push the other one or were you waiting for my answer?